src/main/java/org/minions/devfund/bruno/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,239 @@
+        }
+    }
+
+    /**
+     * Updates the movie rating.
+     *
+     * @param name      String movie.
+     * @param newRating Double new Rating.
+     */
+    public void updateRating(final String name, final double newRating) {
+        Objects.requireNonNull(movieList.stream().filter(movie -> movie.getName().equals(name)).findFirst()
+                .orElse(null)).setRating(newRating);
+    }
+
+    /**
+     * Calculates the average from the List movies.
+     *
+     * @param movies List.
+     * @return average rating.
+     */
+    private double calculateTheAverageMovies(final List<Movie> movies) {
+        return movies.stream().mapToDouble(Movie::getRating).sum() / movies.size();
+    }
+
+    /**
+     * Gets the best Actor.
+     *
+     * @return String name actor.
+     */
+    public String getBestActor() {
+        String bestActor = "";
+        double bestAverage = 0;
+        for (Actor actor : actorList) {
+            if (calculateTheAverageMovies(actor.getMovies()) > bestAverage) {
+                bestAverage = calculateTheAverageMovies(actor.getMovies());
+                bestActor = actor.getName();
+            }
+        }
+        return bestActor;
+    }
+
+    /**
+     * Gets the best movie.
+     *
+     * @return String movie name.
+     */
+    public String getBestMovie() {
+        String bestMovie = "";
+        double majorRating = 0;
+        for (Movie movie : movieList) {
+            if (movie.getRating() > majorRating) {
+                majorRating = movie.getRating();
+                bestMovie = movie.getName();
+            }
+        }
+        return bestMovie;
+    }
+
+    /**
+     * Gets the actor list.
+     *
+     * @return actor List.
+     */
+    public List<Actor> getActorList() {
+        return actorList;
+    }
+
+    /**
+     * Gets the movie list.
+     *
+     * @return movie List.
+     */
+    public List<Movie> getMovieList() {
+        return movieList;
+    }
+
+    /**
+     * Manages the Actors and Movies generates according the movies and rating data base file.
+     */
+    public void main() {
+        List<String> linesMovies = readDataBase(DATABASE_MOVIES_PATH);
+        List<String> linesRating = readDataBase(DATABASE_RATING_PATH);
+        linesMovies.forEach(line -> generateActors(line.split(", ")));
+        linesRating.remove(0);
+        linesRating.forEach(line -> addsTheRatingObtainedFromDB(line.split("\t")));
+    }
+
+    /**
+     * Reads a txt data base file according the path file.
+     *
+     * @param path data base File.
+     * @return a List with the lines obtained from the data base file.
+     */
+    public List<String> readDataBase(final String path) {
+        List<String> lines = new LinkedList<>();
+        Path paths = Paths.get(path);
+        try (Stream<String> stream = Files.lines(paths)) {
+            lines = stream.collect(Collectors.toList());
+        } catch (IOException e) {
+            System.out.println(e);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)